src/test/java/Homework22.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+
+
+
+
+public class Homework22 extends BasePage
+{
+
+    public Homework22(WebDriver givenDriver)
+    {
+        super(givenDriver);
+    }
+    public class PlaylistTests
+    {
+        @Test(dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
+        public void renamePlaylist(String email, String password) {
+
+            String playlistName = "My test playlist";
+
+            LoginPage loginPage = new LoginPage(driver);
+            HomePage homePage = new HomePage(driver);
+
+            loginPage.login();
+
+            homePage.doubleClickPlaylist();
+            homePage.enterNewPlaylistName();
+
+            Assert.assertTrue(homePage.doesPlaylistNameExist(playlistName));
```

Comment:
By the way, I see in this step you are verifying `playlistName `exist. But the playlistName you declared above is `String playlistName = "My test playlist";`. While in step `enterNewPlaylistName()` you are entering another name in `HomePage`class 

![image](https://github.com/testprojp/A47-QA-Automation-Framework/assets/44592718/69fdab0d-f0a8-40d2-8d9e-c480388f739f)

-> Verify will fail because of 2 different values. Please check and update again